test/fhirtypes/ElementDefinition.constrainType.test.ts

```diff
@@ -1207,102 +1207,288 @@ describe('ElementDefinition R5', () => {
+        ];
+        performedActivity.constrainType(onlyRule, fisher);
+        expect(performedActivity.type).toHaveLength(1);
+        expect(performedActivity.type[0]).toEqual(
+          new ElementDefinitionType('CodeableReference').withTargetProfiles(
+            'http://hl7.org/fhir/StructureDefinition/Practitioner',
+            'http://hl7.org/fhir/StructureDefinition/Organization'
+          )
+        );
+        expect(loggerSpy.getAllMessages('error')).toHaveLength(0);
+        expect(loggerSpy.getAllMessages('warn')).toHaveLength(1);
+        expect(loggerSpy.getLastMessage('warn')).toMatch(
+          /CodeableReference\(\) keyword should be used/i
+        );
+      });
+
+      it('should allow a CodeableReference to multiple resource types to be constrained to a reference to a single type', () => {
+        const performedActivity = r5CarePlan.elements.find(
+          e => e.id === 'CarePlan.activity.performedActivity'
+        );
+        const onlyRule = new OnlyRule('activity.performedActivity');
+        onlyRule.types = [{ type: 'Practitioner', isReference: true }];
+        performedActivity.constrainType(onlyRule, fisher);
+        expect(performedActivity.type).toHaveLength(1);
+        expect(performedActivity.type[0]).toEqual(
+          new ElementDefinitionType('CodeableReference').withTargetProfiles(
+            'http://hl7.org/fhir/StructureDefinition/Practitioner'
+          )
+        );
+        expect(loggerSpy.getAllMessages('error')).toHaveLength(0);
+        expect(loggerSpy.getAllMessages('warn')).toHaveLength(1);
+        expect(loggerSpy.getLastMessage('warn')).toMatch(
+          /CodeableReference\(\) keyword should be used/i
+        );
+      });
+
+      it('should allow a CodeableReference to multiple resource types to be constrained to a reference to a single profile', () => {
+        const performedActivity = r5CarePlan.elements.find(
+          e => e.id === 'CarePlan.activity.performedActivity'
+        );
+        const onlyRule = new OnlyRule('activity.performedActivity');
+        onlyRule.types = [
+          { type: 'http://hl7.org/fhir/StructureDefinition/actualgroup', isReference: true }
+        ];
+        performedActivity.constrainType(onlyRule, fisher);
+        expect(performedActivity.type).toHaveLength(1);
+        expect(performedActivity.type[0]).toEqual(
+          new ElementDefinitionType('CodeableReference').withTargetProfiles(
+            'http://hl7.org/fhir/StructureDefinition/actualgroup'
+          )
+        );
+        expect(loggerSpy.getAllMessages('error')).toHaveLength(0);
+        expect(loggerSpy.getAllMessages('warn')).toHaveLength(1);
+        expect(loggerSpy.getLastMessage('warn')).toMatch(
+          /CodeableReference\(\) keyword should be used/i
+        );
+      });
+
+      it('should allow a CodeableReference to multiple resource types to be constrained to a reference to a single profile', () => {
```

Comment:
```suggestion
      it('should allow a CodeableReference to multiple resource types to be constrained to a reference to multiple profiles', () => {
```

Reply:
Updated in [9f88612](https://github.com/FHIR/sushi/pull/1292/commits/9f88612248639230232f7cb9157c6dd3bb37663b)